.github/workflows/ci.yml

```diff
@@ -57,16 +57,38 @@ jobs:
           submodules: 'recursive'
       - name: Build Hardware
         run: |
-          make CFG_OVERRIDE=cfg/snax.hjson \
+          make CFG_OVERRIDE=cfg/snax-mac.hjson \
           -C target/snitch_cluster bin/snitch_cluster.vlt
       - name: Build Software
         run: |
           make -C target/snitch_cluster sw
       - name: Run Tests
         working-directory: target/snitch_cluster
         run: |-
-          ./sw/tests/run.py sw/tests/snax-run.yaml --simulator verilator
-          ./sw/apps/run.py sw/apps/snax-run.yaml --simulator verilator
+          ./sw/tests/run.py sw/tests/snax-mac-run.yaml --simulator verilator
+          ./sw/apps/run.py sw/apps/snax-mac-run.yaml --simulator verilator
+
+  sw-snax-gemm-cluster-vlt:
+    name: Simulate SW on SNAX GEMM Cluster w/ Verilator
+    runs-on: ubuntu-22.04
+    container:
+      image: ghcr.io/pulp-platform/snitch_cluster:ipc-verification
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Build Hardware
+        run: |
+          make CFG_OVERRIDE=cfg/snax-gemm.hjson \
+          -C target/snitch_cluster bin/snitch_cluster.vlt
+      - name: Build Software
+        run: |
+          make -C target/snitch_cluster sw
+      - name: Run Tests
+        working-directory: target/snitch_cluster
+        run: |-
+          ./sw/tests/run.py sw/tests/snax-gemm-run.yaml --simulator verilator
+          ./sw/apps/run.py sw/apps/snax-gemm-run.yaml --simulator verilator
```

Comment:
Ah! hehe okay 😄 I put it because I thought we wanted to also see that the SNAX-GEMM also passes the original tests just in case. It's also quite pre-emptive for the next ones 🔦